dcplib/etl/__init__.py

```diff
@@ -133,15 +134,17 @@ def extract(self, query: dict = None, transformer: callable = None, loader: call
             bundles_complete += self.enqueue_bundle_manifests(percent_complete, bundles_complete, total_bundles,
                                                               f2f_futures, ff_futures, executor)
 
-        for future in concurrent.futures.as_completed(ff_futures):
-            f, bundle_uuid, bundle_version = future.result()
-            self.b2f[bundle_uuid + "." + bundle_version].remove(f["name"])
-            if len(self.b2f[bundle_uuid + "." + bundle_version]) == 0:
-                self.dispatch_callbacks(bundle_uuid, bundle_version, transformer, loader)
-            else:
-                logger.debug("%s: %d files to go", bundle_uuid, len(self.b2f[bundle_uuid + "." + bundle_version]))
-        for bundle_uuid, bundle_version in self.staged_bundles:
-            self.dispatch_callbacks(bundle_uuid, bundle_version, transformer, loader)
+        with dispatch_executor_class(max_workers=max_dispatchers) as executor:
```

Comment:
Is there a performance concern running this with `max_workers=1`?

Reply:
Not sure if I follow. Here, max_workers=1 is the same as no multithreading at all, right?

Reply:
Yea, unnecessary but leaving as is for code simplicity